make/migrations/postgresql/0035_2_1_0_schema.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE blob ADD COLUMN update_time timestamp default CURRENT_TIMESTAMP;
```

Comment:
Try to make it idempotent, such as "IF NOT EXIST"

Reply:
updated